lib/loginUtils.js

```diff
@@ -11,28 +11,31 @@ export async function loadStoreData() {
 export async function lookupClerk(storeId, pin) {
   const currStore = await getStoresById(storeId);
   const clerks = await getClerksByIds(currStore.clerkIds);
-  let errorMsg = 'Invalid PIN';
+  let errorMsg = '';
+
   // No clerk at this store
   if (clerks.length === 0) {
-    return { status: status.NOT_FOUND, record: null, errorMsg };
+    errorMsg = 'No clerk found at this store';
+    return { status: status.NOT_FOUND, record: null, storeName: `${currStore.storeName}`, errorMsg };
   }
-  // Iterate through clerks at this store, and find the one with this password.
-  const matching = clerks.filter(clerk => clerk.pin === pin);
 
+  // Iterate through clerks at this store, and find the one with this password.
+  const matching = clerks.filter((clerk) => clerk.pin === pin);
   if (matching.length === 0) {
     // Incorrect password
-    return { status: status.FOUND, record: null, errorMsg };
+    errorMsg = 'Incorrect password';
```

Comment:
This should say 'Invalid PIN' (probably should update the comment as well); this was before we updated the column names